scripts/run-integration-tests.sh

```diff
@@ -5,6 +5,8 @@ set -Euo pipefail
 trap 'on_error $? $LINENO' ERR
 
 DIR=$(cd "$(dirname "$0")"; pwd)
+INTEGRATION_TEST_DIR="$DIR"/../test/integration-new
```

Comment:
Can we rename the folder?

Reply:
we do have folder called integration. do you have any better name in mind ?

Reply:
Can we merge both the folders instead?